boilerplate/README.md

```diff
@@ -110,6 +113,36 @@ This is a great place to put miscellaneous helpers and utilities. Things like da
 
 **app.tsx** This is the entry point to your app. This is where you will find the main App component which renders the rest of the application.
 
+### ./assets directory
+
+This directory is designed to organize and store various assets, making it easy for you to manage and use them in your application. The assets are further categorized into subdirectories, including `icons` and `images`:
+
+```
+assets
+├── icons
+└── images
+```
+
+**icons**
+This is where your icon assets will live. These icons can be used for buttons, navigation elements, or any other UI components. The recommended format for icons is PNG, but other formats can be used as well.
+
+Ignite comes with a built-in `Icon` component. You can find detailed usage instructions in the [docs](https://github.com/infinitered/ignite/blob/master/docs/Components-Icon.md).
+
+**images**
+This is where your images will live, such as background images, logos, or any other graphics. You can use various formats such as PNG, JPEG, or GIF for your images.
+
+How to use your `icon` or `image` assets:
+
+```
+import { Image } from 'react-native';
```

Comment:
Should we point to the `AutoImage` component from the components dir do you think?

Reply:
Added AutoImage ref to docs

Reply:
@frankcalise added the `AutoImage` ref in the docs